toolkit/tools/imagegen/diskutils/diskutils.go

```diff
@@ -419,6 +459,11 @@ func CreateSinglePartition(diskDevPath string, partitionNumber int, partitionTab
 		return "", err
 	}
 	logger.Log.Debugf("Partprobe -s returned: %s", stdout)
+
+	if partType == "extended" {
```

Comment:
Could you add a comment explaining why we bail out early here?

Reply:
A debug log could do that and we'd have more info in the logs as well at the same time.

Reply:
This logic is not needed. Have removed this piece of code.